web/src/i18n/locales/en/apd/apd.yaml

```diff
@@ -44,13 +44,13 @@ hie:
       [/ol]
     helpText: >-
       CMS recommends that states connect with regional CMS HITECH contacts prior to
-      developing HIE funding request. HIE funding requests are complex. Early
+      developing an HIE funding request. HIE funding requests are complex. Early
```

Comment:
"a HIE"? Did some searching, using "a" before H when the sound is a consonant and use "an" before H when the sound is a vowel sound...

Reply:
Depends on whether you pronounce the initialism (like "hi" or "hee" or something), or if you pronounce the letters ("eich eye ee").  In the former case, it'd be "a HIE" because the word begins with a hard sound; but in the latter, it'd be "an HIE" because the pronunciation of the letter "H" begins with a soft sound.  This is one of those quirky areas of the English language because the written form is dictated by the pronunciation, which of course varies from person to person.

And I don't feel bad about stepping into James's turf because James's watch is ended.